nss-day-cohort-60/rare-client-tenacious-tators
# Mp stylin2

Closes Ticket #


# Description of Proposed Changes

Added hover effects to category links in cat manager and pointer cursor to trashcan and edit icons

---


# Steps to Test

1. Fetch the `MP-Stylin2` branch and switch to it
2. Start your debugger and npm start your client
3. Navigate to category manager. Links should change color on hover and cursor should be a pointer finger over edit and trashcan icons
